apps: charts to ghcr (#1912)
elastisys/compliantkubernetes-apps: helmfile.d/charts/README.md

# GHCR

> As part of our effort to make our helmcharts available for public and internal consumption we have decided to publicise some of them in our "Github Container Registry".

Reviewer: ```suggestion
As part of our effort to make our Helm charts available for public and internal consumption we have decided to publish some of them on Github Container Registry.
```
